ext/scheduler/airflow2/resources/base_dag.py

```diff
@@ -53,6 +53,38 @@
     {{ base $baseTaskSchema.SecretPath | quote }}
 )
 {{- end }}
+
```

Comment:
All of this should be added in `ext/scheduler/airflow/...` as well to support airflow 1.

Reply:
Oh right, I haven't put it. Will update it.

Reply:
it's updated